wgsl/media-type-registration.txt

```diff
@@ -0,0 +1,65 @@
+Type name: text
+
+Subtype name: wgsl
+
+Required parameters: N/A
+
+Optional parameters: None
+
+Encoding considerations: binary
```

Comment:
"binary" is used here. The other options are:
7bit: for 7bit text
8bit: for 8bit text separated by CR/LF.  But UTF-8 doesn't fit that.
binary: all 8 bits in each octet must be preserved.  text/javascript uses "binary"
framed:  For framed binary data